wiki/Tournaments/id.md

```diff
@@ -1,7 +1,3 @@
----
-outdated_translation: true
----
```

Comment:
> > Hmm, iirc there are more things that need updating here than just the listing tho. As it stands the `id.md` is already lagging \*very\* far behind the original `en.md` so yeah...

@Nivalyx if the translation is still outdated even after these changes, then this marker shouldn't be removed

Reply:
o yes true

@FazlyMR

Reply:
>yup, i think it's fine to push this for now as is and update the other details later as a follow-up in a different pr

I'm still not sure what that is lacking in the Indonesian translation. I have checked it with the current English version and it seems to not be lagging very far aside from needing to update the tournament listing.

Reply:
mainly some missing lines every here and there, for example this line is still not present in the current Indonesian translation 
![image](https://user-images.githubusercontent.com/36564236/232844051-bfbdb984-44d0-4181-ae3f-0ce867a459c2.png)

Reply:
Ohh, I didn't see that. Ok then. See my next comment for my suggestion on that.